.github/workflows/build_rpm.yaml

```diff
@@ -0,0 +1,174 @@
+          ref: 'ci/create-rpm-package'
+          path: 'packaging'
+
+      - name: Export good OS version
+        run: |
+          echo "OS=$(echo ${{ matrix.os }} | awk -F'[/:]' '{print $(NF-1) "" $NF}')" >> $GITHUB_ENV
+
+      - name: Install RPM on Fedora
+        if: contains(matrix.os, 'fedora')
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs  | sh -s -- -y
+          export PATH=$PATH:$HOME/.cargo/bin/
+          sudo dnf install rpm-build -y
+          sudo dnf install cargo-rpm-macros -y
+          sudo dnf install openssl-devel -y
+
+      - name: Install RPM on other than Fedora
+        if: ${{ ! contains(matrix.os, 'fedora') }}
+        run: |
+          mkdir -p /opt/rustup
+          mkdir -p /opt/cargo
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs  | RUSTUP_HOME=/opt/rustup CARGO_HOME=/opt/cargo sh -s -- -y
+          echo 'export PATH=$PATH:/opt/rustup:/opt/cargo/bin' >> ~/.bashrc && source ~/.bashrc
+          echo 'export PATH=$PATH:/opt/rustup:/opt/cargo/bin' >> $GITHUB_PATH
+          packages=("unzip" "rpm-build" "openssl-devel" "curl" "gcc")
+          for package in "${packages[@]}"; do
+            if ! yum list installed "$package" >/dev/null 2>&1; then
+              yum install -y --allowerasing "$package"
+            fi
+          done
+          rustup default stable
+
+      - name: Setup for alumet
+        run: |
+          mkdir -p alumet
+          mkdir -p /etc/alumet  // TODO Change on merge !!
+
+      - name: Checkout alumet repo
+        uses: actions/checkout@v4
+        with:
+          # repository: alumet-dev/alumet
+          repository: AngeCyp/alumet-testing  # TODO Change on merge
+          path: "alumet-agent"
+          ref: '${{ inputs.tag }}'
+
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+          cache-targets: "alumet"
+
+      - name: get Spec settings
+        id: get-spec-settings
+        env:
+          build_dir: "$(pwd)/build"
+        run: |
+          mkdir -p ${{ env.build_dir }}/SOURCES
+          tar --exclude=".[^/]*" --exclude="./build" --transform 's|.|alumet-agent|' -zcvf ${{ env.build_dir }}/SOURCES/alumet-agent.tar.gz .
+          rpmbuild -ba -vv --define "_topdir ${{ env.build_dir }}" --define "release ${{ inputs.release-version }}" --define "osr ${{ env.OS }}" --define "version ${{ inputs.build-version }}" --define "arch ${{ inputs.target-architecture }}" packaging/rpm/specs/alumet.spec
+
+      - name: Upload artifact alumet-agent
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.name }}-agent-${{ inputs.build-version }}-${{ inputs.release-version }}.${{ env.OS }}.${{ inputs.target-architecture }}.rpm
+          path: |
+            ./build/RPMS/${{ inputs.target-architecture }}/${{ env.name }}-agent-${{ inputs.build-version }}-${{ inputs.release-version }}.${{ env.OS }}.${{ inputs.target-architecture }}.rpm
+
+  test-rpm:
```

Comment:
`testing` to use same kind of terminology as `packaging`. imo no need to precise `rpm` here as we're inside build_rpm workflow